admin/src/components/util/modal/modal-dialog.jsx

```diff
@@ -0,0 +1,62 @@
+import { React } from "react";
+import { Modal, Button } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+/**
+ * @param {object} props
+ * The props.
+ * @param {string} props.text
+ * The modal text
+ * @param {string} props.title
+ * The modal title
+ * @param {string} props.acceptText
+ * The text for the acceptbutton
+ * @param {string} props.declineText
+ * The text for the declinebutton
+ * @param {Function} props.onClose
+ * The callback for close.
+ * @param {Function} props.handleAccept
+ * The callback for accept.
+ *   @returns {object}
+ *   The TagList
+ */
+function ModalDialog({
+  text,
+  title,
+  acceptText,
+  declineText,
+  onClose,
+  handleAccept,
+}) {
+  return (
+    <div className="modal-container">
+      <Modal.Dialog>
```

Comment:
It might be me, but should these dot-notation tags have been `ModalHeader`? To me it looks strange...